waifus/kurisu makise.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+	"type" : 0,
+	"japanese-name" : "牧瀬 紅莉栖",
+	"english-name" : "Kurisu Makise",
+	"year" : 1992,
+	"month" : 7,
+	"day-of-month" : 25,
+	"age-in-show": 18,
+	"image" : "https://steins-gate.fandom.com/wiki/Kurisu_Makise?file=Kurisu%20profile.png",
```

Comment:
link doesn't go to the image 
![image](https://user-images.githubusercontent.com/16181109/67109578-cee8c980-f19e-11e9-8ed5-416d5383d06b.png)
use this one https://vignette.wikia.nocookie.net/steins-gate/images/4/4a/Kurisu_Full_profile.png